File: twiml/voice/stream/stream-1/stream-1.6.x.cs
using System;
using Twilio.TwiML;
using Twilio.TwiML.Voice;


class Example
{
    static void Main()
    {
        var response = new VoiceResponse();
        var start = new Start();
        start.Stream(name: "Example Audio Stream", url: "wss://example.com/audiostream");
        response.Append(start);
        response.Say("The stream has started.");

        Console.WriteLine(response.ToString());
    }
}

File: twiml/voice/stream/stream-1/stream-1.6.x.py
from twilio.twiml.voice_response import VoiceResponse, Say, Start, Stream

response = VoiceResponse()
start = Start()
start.stream(name='Example Audio Stream', url='wss://example.com/audiostream')
response.append(start)
response.say('The stream has started.')

print(response)
